Resources/Prototypes/XenoArch/Effects/normal_effects.yml

```diff
@@ -652,11 +674,214 @@
     maxIntensity: 50
 
 - type: artifactEffect
-  id: EffectPortal
-  targetDepth: 3
-  effectHint: artifact-effect-hint-displacement
+  id: EffectMajorCashSpawn
+  targetDepth: 4
+  effectHint: artifact-effect-hint-creation
   components:
-  - type: PortalArtifact
+  - type: SpawnArtifact
+    maxSpawns: 10
+    spawns:
+    - id: SpaceCash100
+      maxAmount: 5
+      prob: 0.75
+    - id: SpaceCash1000
+      maxAmount: 2
+      prob: 0.5
+    - id: SpaceCash5000
+      prob: 0.25
+    - id: SpaceCash10000
+      prob: 0.1
+    - id: SpaceCash50000
```

Comment:
There is no prototype for `SpaceCash50000`. The list of valid prototypes for SpaceCash are in [`Resources/Prototypes/Entities/Objects/Misc/space_cash.yml`](https://github.com/impstation/imp-station-14/blob/master/Resources/Prototypes/Entities/Objects/Misc/space_cash.yml). You should either use one of the existing prototypes, or create a new one for `SpaceCash50000`.

Reply:
Fixed in commit 535a688